src/cltkv1/wrappers/__init__.py

```diff
@@ -1,11 +1,13 @@
 """Init for `cltkv1.wrappers`."""
 
-from cltkv1.utils.data_types import Doc, MultiProcess, Word
+import copy
+
+from cltkv1.utils.data_types import Doc, Process, Word
 
 from .stanford import StanfordNLPWrapper
 
 
-class StanfordNLPProcess(MultiProcess):
+class StanfordNLPProcess(Process):
```

Comment:
OK, this is your answer to my above about MultiProcess. Let's go with this for now and then abstract up to MultiProcess if/when necessary.